src/Filter/modules/SingleSelectFilter/SingleSelectFilter.tsx

```diff
@@ -46,9 +45,9 @@ const SingleSelectFilter = (
             filterOptions.unshift({ label: selectPlaceholderLabel || 'Select', value: '' });
         }
 
-        const handleUpdate = (option: SelectOverlayOption | null) => {
-            update(option?.value);
-        };
+                const handleUpdate = (option: { currentTarget: { value: string | number | null | undefined } }) => {
+                    update(option.currentTarget.value);
```

Comment:
fyi: it's more typical to consume `target` (the triggering/root element) vs `currentTarget` (the element that an event listener is attached to -- which may not be the root due to event bubbling) from events. (e.g. `event.target.value`). Since [in this case](https://github.com/ToyotaResearchInstitute/lakefront/blob/edc58fb060c34f40ebc93ac8826b0d4355bf72d2/src/Select/SelectOverlay.tsx#L18) both are the same value, I'm not opposed to this implementation. More stating for reference.